opencti-platform/opencti-graphql/src/manager/ruleManager.ts

```diff
@@ -294,11 +304,13 @@ const initRuleManager = () => {
       running = true;
       const ruleManager = await getInitRuleManager();
       const { lastEventId } = ruleManager;
```

Comment:
should remove the previous from elastic and put it in redis, so we don't retain old state values in elastic which can be confusing during investigation